packages/lit-task-auto-top-up/src/types/types.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod';
```

Comment:
Zod <-> Rust inter-op...might be worth looking into for single-source-of-truth with the node Rust codebase

https://www.reddit.com/r/rust/comments/17dyjyd/i_made_a_crate_for_bridging_types_between_rust/
https://github.com/spencerkohan/type_reflect